ignite/pkg/chaincmd/runner/chain.go

```diff
@@ -45,6 +45,15 @@ func NewKV(key, value string) KV {
 
 var gentxRe = regexp.MustCompile(`(?m)"(.+?)"`)
 
+func (r Runner) InPlace(ctx context.Context, newChainID, newOperatorAddress string, options ...chaincmd.InPlaceOption) error {
+	fmt.Println("Press Ctrl + C to stop the running testnet process.")
```

Comment:
Are we able to as well display the chain logs?

Reply:
Is this necessary, since other ignite commands do not show the chain log, such as `ignite chain serve`

Reply:
You can let them display it with `--verbose`. Here it would be useful to support that as well yes.

Reply:
cc @likesToEatFish